### ISSUE TITLE
Plugin that sorts defined array

### ISSUE THREAD

**Issue by atompie** [Tracardi/tracardi#732]
**Is your feature request related to a problem? Please describe.**
Plugin that sorts (ascending, descending) a referenced array in plugin config. This plugin is very simple in terms of python. The challenge is to add it as plugin. This is also very simple if you read this http://docs.tracardi.com/plugins/tutorial/part1/ and http://docs.tracardi.com/plugins/tutorial/part2

**Describe the solution you'd like**
User defines in plugin configuration what array needs to be sorted and if it is ascending, descending sorting. 

Example of configuration 

```json
{
   "data": "event@properties.list_of_something",
   "direction": "asc"
}
```

Input:
Workflow payload

Output:
Sorted array

**Additional context**
You will need to setup the dev environment. Here is the tutorial how to do it. Also to understand how the plugins work please read http://docs.tracardi.com/plugins/tutorial/part1/ and http://docs.tracardi.com/plugins/tutorial/part2/ . The next part is o its way so you can extend your knowledge about Tracardi plugins.

**Comment by tanayseven** [comment]
Will pick this issue up since it is a good first issue. Do I need to know a lot about Tracardi before being able to contribute? Or is basic knowledge and local setup good enough for contributing to this?

**Comment by atompie** [comment]
@tanayseven You do not have to know much about tracardi if your read the tutorials you should be ready to go.

**Comment by tanayseven** [comment]
Yes, I would like to be assigned. So you're saying I can read the tutorials from the two links you've pasted, right?

**Comment by MANISH-LAB** [comment]
assign me

**Comment by atompie** [comment]
@tanayseven This should be enough.

**Comment by atompie** [comment]
@tanayseven Do you need some help with this issue?

**Comment by tanayseven** [comment]
Not right now, thanks. I'll let you know if I need any help

**Comment by tanayseven** [comment]
A bit busy for a couple of days, I'll continue working on this tomorrow

**Comment by tanayseven** [comment]
I had a few doubts around the implementation, but I thought raising a pull request would be the best way to get them cleared. So can you review the pull request and then we can discuss whatever we're missing?

**Comment by tanayseven** [comment]
This PR is by no means complete and I've raised it only to make the discussion easier from my side

**Comment by atompie** [comment]
@tanayseven sure. I am away from my keyboard till Monday. As soon as I get back I will take a look at your PR.

**Comment by tanayseven** [comment]
Made the changes that you asked me for. But I have one doubt, I've posted a comment on the PR

**Comment by shahad1307** [comment]
@atompie Hello I'm a student in CMU and I have a project. I'm interested in working on this issue. Looks doable could do assign me?

**Comment by atompie** [comment]
@shahad1307 Sorry this one is already done, but not closed. Closing
